ci/jobs/rebase_analysis.yaml

```diff
@@ -0,0 +1,51 @@
+# This Jenkins job runs a script which analyzes the gap between upstream and downstream by querying
+# Redmine and Bugzilla. See https://github.com/pulp/pulp_packaging/ci/redmine_bugzilla.py
+
+- job-template:
+    name: 'rebase-analysis'
+    defaults: ci-workflow-runtest
+    node: 'fedora-np'
+    properties:
+        - dev-ownership
+    scm:
+        - git:
+            url: 'https://github.com/pulp/pulp_packaging.git'
+            branches:
+                - origin/master
+            basedir: pulp_packaging
+            skip-tag: true
+            wipe-workspace: false
+    triggers:
+        - timed: "@midnight"
+    wrappers:
+        - credentials-binding:
+            - file:
+                credential-id: 099bc04b-80b6-4cd6-ad1d-11908641f539
+                variable: REDMINE_BUGZILLA_CONF
+        - timeout:
+            # Timeout in minutes
+            timeout: 60
+            timeout-var: 'BUILD_TIMEOUT'
+            fail: true
+    builders:
+        - shell: |
+            #!/bin/bash
+            git config --global user.email "pulp-infra@redhat.com"
+            git config --global user.name "pulpbot"
+            set -x
+            env
+
+            cd $WORKSPACE/
+
+            sudo pip install python-bugzilla python-redmine certifi
+
+            # Here due to InsecurePlatformWarning
+            # https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning
+            #sudo pip install ndg-httpsclient pyasn1 pyopenssl urllib3
```

Comment:
I was leaving this in because I thought I would need it after I pushed the jobs, but I can just bring it back from version control if that happens. I am removing it with next force push.